src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -85,11 +159,14 @@ export const StakingOpportunities = ({ assetId, opportunities }: StakingOpportun
     e.stopPropagation()
   }
 
-  const handleStakedClick = () => {
-    cosmosStaking.open({ assetId: 'cosmos:cosmoshub-4/slip44:118' })
+  const handleStakedClick = (values: Row<ActiveStakingOpportunity>) => {
+    cosmosStaking.open({
+      assetId: 'cosmos:cosmoshub-4/slip44:118',
```

Comment:
`<StakingOpportunities />` is already passed an `assetId` as a prop ([ref](https://github.com/shapeshift/web/pull/1375/files#diff-96dc327eb19b724e0fcbc22953cd63b9fe03c21cc27ac816ddb6024df6c6b658R77)), let's reuse it here

Reply:
fixed